### DISCUSSION TITLE
Latency Concerns with AWS Lambda Web Adapter to stream responses

### DISCUSSION THREAD

**Discussion by tobixoxo** [awslabs/aws-lambda-web-adapter#450] [Q&A] [answered]
Hi, this question is in context with response streaming examples that are provided.
My aim was to make a pure backend api which streams responses in python, for which the fast api example proved to be a valuable reference.
I was wondering if there is any additional latency which might arise as here we are using web adapter along with the lambda function. 
Any insight would be valuable to me, Thanks!

**Comment by bnusunny** [comment, marked as answer]
We discussed in [this issue](https://github.com/awslabs/aws-lambda-web-adapter/issues/283).

**Reply by tobixoxo** [reply]
thank you
